doc/en/doctest.rst

```diff
@@ -303,3 +294,18 @@ docstring.
 
     * Text files: the functions will skip/xfail the checks for the rest of the entire
       file.
+
+
+Alternatives
+------------
+
+While the built-in pytest support provides a good set of functionalities for using
+doctests, if you use them extensively you might be interested in those external packages
+which add many more features, and include pytest integration:
+
+* `pytest-doctestplus <https://github.com/astropy/pytest-doctestplus>`__: provides
```

Comment:
@astrofrog and @saimn , any objection to this mention here?

Reply:
No, sounds good!